server/lib/api/users.js

```diff
@@ -97,3 +106,33 @@ async function resetPassword(ctx, next) {
     ctx.status = 204
   })
 }
+
+async function verifyEmail(ctx, next) {
```

Comment:
I went to do this, but I have no idea if there's any rationale behind choosing the rate/burst/window values. All of these values seem like they're pulled out of a hat to me.